tensorbay/client/cloud_storage.py

```diff
@@ -3,11 +3,7 @@
 # Copyright 2021 Graviti. Licensed under MIT License.
 #
 
-"""Class CloudClient.
-
-The :class:`CloudClient` defines the initial client to interact between local and cloud platform.
-
-"""
+"""The initial client to interact between local and cloud platform."""
```

Comment:
This is just the docstring of `CloudClient` without `StorageConfig`.  It is better to have a more general summary, which can be obtained by referring to our documentation.